### ISSUE TITLE
Issue with negative bounds and slider fill-in area when Slider is Disabled

### ISSUE THREAD

**Issue by blzone** [seiyria/bootstrap-slider#546]
Issues
=============
**NOTE:** Before filing a new issue, please search through the open issues list and check to see if an existing issue already exists that describes your problem. If so, leave a comment within that issue with the checklist items described below.

Please make sure you include the following in your issue report where appropriate:

- [ ] JSFiddle (or an equivalent such as CodePen, Plunker, etc) with the an example demonstrating the bug
- [x] Instructions to replicate the bug in the Fiddle

**Comment by seiyria** [comment]
Please actually fill in the template.

**Comment by blzone** [comment]
Hi,

I am working on an application where I need the ability to enable and disable the slider.  If I specify positive numbers for the bounds of the slider (i.e. 0 - 10, 1 - 30, 50 - 500), and I then set the slider to disabled ( using sliderId.slider('setAttribute', undefined)), everything works. The slider, as expected, is shown as a hallowed out line (I also disable the tooltip).  If however I have specified a negative bound (say -3 to +10), when I set the slider to disabled using: sliderId.setAttribute('setAttribute', 'enabled', false) the slider seems to fill in up to some approximate value almost as if the user has set a value.

Is there a bug here, or am I not using the slider API correctly?

Sample code (uses JQuery):

panel.find('.enableSlider').on('click', function(e) {
  var sliderId= panel.find('.slideStyle');
  sliderId.slider('setAttribute', 'enabled', true);
  sliderId.slider('setAttrbute', 'tooltip', 'show');
  sliderId.slider('setAttribute', 'value', someValue..); // I actually set it to the midpoint of the bounds
  sliderId.slider('refresh');

  panel.find.find('.min-slider-handle').each(function() {
     $(this).show();
  });
});
panel.find('.disable-slider-button').on('click', function(e) {
  var sliderId= panel.find('.slideStyle');
  sliderId.slider('setAttribute', 'enabled', false);
  sliderId.slider('setAttribute', 'tooltip', 'hide');
  sliderId.slider('setAttribute', 'value', undefined);  <--- is this the correct way to have no value set?
  sliderId.slider('refresh');

  $(this).addClass('disabled');
});
